src/__tests__/a11yAPI.test.tsx

```diff
@@ -396,31 +392,6 @@ test('getAllByA11yValue, queryAllByA11yValue, findAllByA11yValue', async () => {
   await expect(findAllByA11yValue({ max: 60 })).resolves.toHaveLength(2);
 });
 
-test('a11y label queries have aliases', () => {
-  const {
-    getByA11yLabel,
-    getByLabelText,
-    queryByA11yLabel,
-    queryByLabelText,
-    findByA11yLabel,
-    findByLabelText,
-    getAllByA11yLabel,
-    getAllByLabelText,
-    queryAllByA11yLabel,
-    queryAllByLabelText,
-    findAllByA11yLabel,
-    findAllByLabelText,
-  } = render(<Section />);
-
-  // Assert that query aliases are referencing the same function
-  expect(getByA11yLabel).toBe(getByLabelText);
-  expect(queryByA11yLabel).toBe(queryByLabelText);
-  expect(findByA11yLabel).toBe(findByLabelText);
-  expect(getAllByA11yLabel).toBe(getAllByLabelText);
-  expect(queryAllByA11yLabel).toBe(queryAllByLabelText);
-  expect(findAllByA11yLabel).toBe(findAllByLabelText);
-});
```

Comment:
While we may discuss about usefulness of this test, why it gets removed with this PR? :)

Reply:
This test just checks that aliases work and we removed them

https://github.com/callstack/react-native-testing-library/pull/881#discussion_r768432588